CONTRIBUTING.md

```diff
@@ -28,6 +28,12 @@ You generally only need to submit a CLA once, so if you've already submitted one
 (even if it was for a different project), you probably don't need to do it
 again.
 
+## Branching Model
+
+This repository uses the [Gitflow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) branching model.
```

Comment:
Can you remove the space after the link and put `branching model.` on a new line?

Reply:
Done.